eventarc/audit-storage/test/runner.sh

```diff
@@ -39,8 +46,6 @@ function cleanup {
 }
 trap cleanup EXIT
 
-# TODO: Perform authentication inside the test.
-export ID_TOKEN=$(gcloud auth print-identity-token)
```

Comment:
You can generate an ID token from `auth` with:

```yaml
steps: 
- uses: 'google-github-actions/auth@v0'
  with:
    # ... existing values
    token_format: 'id_token'
    id_token_audience: 'TODO'
    id_token_include_email: true
```

For Cloud Run, the audience must be the _full_ URL of the Cloud Run service (e.g. `https://my-cloud-run-service.run.app/`).